src/options.cairo

```diff
@@ -200,18 +206,17 @@ mod Options {
         res
     }
 
-    fn run_add_0911_1611_options() {
+    fn run_add_0501_options() {
         let mut state = Governance::unsafe_new_contract_state();
         assert(!state.proposal_initializer_run.read(36), 'prop already initialized');
 
         state.proposal_initializer_run.write(36, true);
```

Comment:
You need to update the proposal ID here

Reply:
This needs to be any not-yet-used-for-initializer prop ID, 42 was last, but nice to have this align with the planned prop_id.

Reply:
used proposal_initializer_run 44 since that is the assumed proposal ID, or should 43 be used instead?

Reply:
44 ok